pkg/core/kubectl_client.go

```diff
@@ -17,14 +17,14 @@
 package core
 
 import (
+	"github.com/projectriff/riff/pkg/crd"
 	"github.com/projectriff/riff/pkg/kubectl"
 	"k8s.io/client-go/kubernetes"
 )
 
 type KubectlClient interface {
-	SystemInstall(manifests map[string]*Manifest, options SystemInstallOptions) (bool, error)
 	SystemUninstall(options SystemUninstallOptions) (bool, error)
-	NamespaceInit(manifests map[string]*Manifest, options NamespaceInitOptions) error
+	NamespaceInit(manifests map[string]*crd.Manifest, options NamespaceInitOptions) error
```

Comment:
Why does NamespaceInit accept manifests? I would expect it to read the manifest from the cluster and install that into a namespace.